resource-management/test/integration/service-api/basic_test.go

```diff
@@ -0,0 +1,162 @@
+/*
+Copyright 2022 Authors of Global Resource Service.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package service_api
+
+import (
+	"strconv"
+	"sync"
+	"testing"
+	"time"
+
+	"global-resource-service/resource-management/pkg/clientSdk/rmsclient"
+	utilruntime "global-resource-service/resource-management/pkg/clientSdk/util/runtime"
+	"global-resource-service/resource-management/pkg/common-lib/types/event"
+
+	"github.com/stretchr/testify/assert"
+	"k8s.io/klog/v2"
+)
+
+func TestRegisterClient(t *testing.T) {
```

Comment:
No need to test RegisterClient individually

Reply:
deleted